splice/web/api/s3_common.py

```diff
@@ -0,0 +1,27 @@
+MIME_EXTENSIONS = {
+    "png": "image/png",
+    "gif": "image/gif",
+    "jpeg": "image/jpg",
+    "svg": "image/svg+xml",
+    "html": "text/html",
+    "zip": "application/zip",
+    "xpi": "application/x-xpinstall",
+    "css": "text/css",
+    "js": "application/javascript"
+}
+
+
+def setup_s3(bucket="bucket"):
+    from splice.environment import Environment
+    from boto.s3.cors import CORSConfiguration
+
+    env = Environment.instance()
+    bucket = env.s3.get_bucket(env.config.S3[bucket])
+    cors = CORSConfiguration()
+    cors.add_rule("GET", "*", allowed_header="*")
+    bucket.set_cors(cors)
+    headers = {
+        'Cache-Control': 'public, max-age=31536000',
```

Comment:
We actually don't want that max-age for html pages. This may be fine for other resources.

Can we set a max-age to 24h for html files?